news/2025/2025-08-30-osumania-4k-world-cup-2025-round-of-32-recap.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title: "osu!mania 4K World Cup 2025: Round of 32 Recap"
+date: 2025-08-30 17:00:00 +0000
+---
+
+Glory awaits! The osu!mania 4K World Cup bracket stages begin with a loud bang.
+
+![](/wiki/shared/news/2025-08-30-osumania-4k-world-cup-2025-round-of-32-recap/banner.jpg)
+
+This tournament couldn't have started stronger! We got a mix of upsets, close victories, and stellar performances from our top teams all jam-packed into the very, very first round of the tournament. Whether this is an omen of what's to come, that's on future rounds to decide.
+
+On a different note, for those not yet aware, the [MWC 4K 2025 support banners](https://osu.ppy.sh/store/products/1513) have gone up for sale! Those who wish to offer some love to their favourite teams, you can now do so while proudly showing it on your osu! profile.
+
+With the introductions out of the way, feel free to check out this week's video recap brought to you by the hands of [Dynascape](https://osu.ppy.sh/users/8784587), [SunApple](https://osu.ppy.sh/users/11817622), [Normiplier](https://osu.ppy.sh/users/10069850), and [akace100](https://osu.ppy.sh/users/9308128).
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/ld7rUnL9uf8" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Mappool recap
+
+[**Download the pool here! (104 MB)**](https://osu.ppy.sh/beatmaps/packs/P295)
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2542835853&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+![Round of 32 mappool table](/wiki/shared/news/2025-08-30-osumania-4k-world-cup-2025-round-of-32-recap/mappool-ro32.jpg)
+
+With Qualifiers coming to a close, we're finally brought to our first round (of 32). Taking a different approach from the 7-stage structure from the Qualifiers, we've got our hands on the first full pool of the tournament with 6 rice slots, 3 hybrid slots, 3 long note slots, and one tiebreaker. With that said, here are some of the most notable highlights from the round!
+
+The best way to start things off is with [Taishi - Distant Relatives (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2421252#mania/5267303) (HB3) by ::{ flag=DZ }:: [Scotty](https://osu.ppy.sh/users/11085809) and ::{ flag=HK }:: [Alptraum](https://osu.ppy.sh/users/26496648)! With the removal of SV, Round of 32 started with 3 hybrid picks instead of two. And as it is to be expected, HB3 had the most difficult patterning out of all three. With the slot always having a technical focus, you can be sure to expect many dump heavy patterns, intertwining both rice and long note patterns in a rather seamless manner.
```

Comment:
```suggestion
The best way to start things off is with [Taishi - Distant Relatives (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2421252#mania/5267303) (HB3) by ::{ flag=DZ }:: [Scotty](https://osu.ppy.sh/users/11085809) and ::{ flag=HK }:: [Alptraum](https://osu.ppy.sh/users/26496648)! With the removal of SV, Round of 32 started with three hybrid picks instead of two. And as it is to be expected, HB3 had the most difficult patterning out of all three. With the slot always having a technical focus, you can be sure to expect many dump-heavy patterns, intertwining both rice and long note patterns in a rather seamless manner.
```